templates/react-ts/src/main.tsx

```diff
@@ -1,10 +1,9 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./App";
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+import App from './App';
 
-ReactDOM.render(
```

Comment:
ReactDOM.render is deprecated.
This file follow the file generated by the create-vite command.

https://github.com/vitejs/vite/blob/main/packages/create-vite/template-react-ts/src/main.tsx